src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HearingsHelper.java

```diff
@@ -83,6 +88,105 @@ public static List<String> hearingMidEventValidation(CaseData caseData) {
         return errors;
     }
 
+    public static void setEtInitialConsiderationListedHearingType(CaseData caseData) {
+        HearingType earliestListedHearing = getEarliestListedHearingType(caseData.getHearingCollection());
+
+        if (earliestListedHearing == null) {
+            log.info("No listed hearings found for case: {} to set EtInitialConsiderationListedHearingType",
+                    caseData.getEthosCaseReference());
+            return;
+        }
+
+        if (caseData.getEtICHearingListedAnswers() == null) {
+            caseData.setEtICHearingListedAnswers(new EtICHearingListedAnswers());
+        }
+
+        caseData.getEtICHearingListedAnswers().setEtInitialConsiderationListedHearingType(
+                earliestListedHearing.getHearingType());
+    }
+
+    public static HearingType getEarliestListedHearingType(List<HearingTypeItem> hearingCollection) {
+        if (CollectionUtils.isEmpty(hearingCollection)) {
+            return null;
+        }
+
+        return hearingCollection.stream()
+                .filter(hearingTypeItem -> hearingTypeItem != null
+                        && hearingTypeItem.getValue() != null)
+                .map(HearingTypeItem::getValue)
+                .filter(hearing -> hearing.getHearingDateCollection() != null
+                        && !hearing.getHearingDateCollection().isEmpty()
+                        && hearing.getHearingDateCollection().stream().anyMatch(HearingsHelper::isListedHearing))
+                .map(hearing -> new Object[] { hearing,
+                        getEarliestListedFutureHearingDate(hearing.getHearingDateCollection()).orElse(null) })
+                .filter(arr -> arr[1] != null)
+                .min(Comparator.comparing(arr -> (java.time.LocalDate) arr[1]))
+                .map(arr -> (HearingType) arr[0])
+                .orElse(null);
```

Comment:
Couple points from me with this though personal preference on some
- Replace the double check on the filter with CollectionUtils.isNotEmpty()
- Adjusted the spacing just to be consistent with how it is elsewhere
- Rather than creating a generic object and then casting the object, I replaced it with a Map so there's no object casting involved

```suggestion
        return hearingCollection.stream()
            .filter(hearingTypeItem -> hearingTypeItem != null
                    && hearingTypeItem.getValue() != null)
            .map(HearingTypeItem::getValue)
            .filter(hearing -> CollectionUtils.isNotEmpty(hearing.getHearingDateCollection())
                    && hearing.getHearingDateCollection().stream().anyMatch(HearingsHelper::isListedHearing))
            .map(hearing -> Map.entry(
                hearing, 
                getEarliestListedFutureHearingDate(hearing.getHearingDateCollection())))
            .filter(entry -> entry.getValue().isPresent())
            .min(Comparator.comparing(entry -> entry.getValue().orElse(null)))
            .map(Map.Entry::getKey)
            .orElse(null);
```